.github/workflows/CODEOWNERS

```
@@ -0,0 +1,2 @@
+#This file defines the front end team as owners of this repo on Github so the auto assignment will work properly
+* @input-output-hk/frontend
```

Review from phone and I couldn't read the doc, apologies if it's wrong. 
If that's supposed to be the name of the team it should then be
```suggestion
* @input-output-hk/certification/frontend
```

---

It is correct. GitHub assigns names to subteams in a confusing way. Here's a Loom videos that explains why this name added is correct. https://www.loom.com/share/3202d2be9d054269be79a80174b45583

---

Thanks! Strange behaviour, I don't see how it can scale if multiple teams have subteams with the same name so maybe, we'll need to add some unique identifier to our subteam name

---

Yes, I was thinking the same thing yesterday. I think something like certification-frontend could work.